src/Modules/Backend/assets/Backend/webpack/js/Backend.js

```diff
@@ -50,7 +52,15 @@ export class Backend {
     this.tableSequenceDragAndDrop = new TableSequenceDragAndDrop()
     this.session = new Session()
     this.ajaxContentEditable = new AjaxContentEditable(this.locale)
-    this.initToasts = new InitBsToasts()
+
+    // init media selector vue component
+    const vueApp = document.querySelector('[data-role="vue-app"]')
+    if (vueApp) {
+      const app = createApp()
+      // global component
+      app.component('TestComponent', TestComponent)
```

Comment:
Is this test component needed? Or is this some debug-code for the media-library?